### PR TITLE
feature: ping metrics plugin

### PR DESCRIPTION
**Pull request by roobert**

### CODE REVIEW

File: bin/metrics-ping.rb [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+#
+# NOTES:
+#
+# LICENSE:
+#   Copyright 2015 Rob Wilson <roobert@gmail.com>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'rubygems' if RUBY_VERSION < '1.9.0'
```
Comment (tas50) [inline bin/metrics-ping.rb]: There's no need for this line anymore since we don't support Ruby 1.8.7

File: bin/metrics-ping.rb [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  def ping
+    @ping, @status = Open3.capture2e("ping -W#{config[:timeout]} -c#{config[:count]} #{config[:host]}")
+  end
+
+  def validate
+    critical "ping error: (#{status}): #{@ping}" if @status != 0
+  end
+
+  def run
+    ping; validate; output; exit 0
```
Comment (tas50) [inline bin/metrics-ping.rb]: These should be split out onto multiple lines

**Comment by tas50** [comment]
Can you squash this pull request into a single check and update the pull request name to reflect the feature that's getting added here. Other than those small comments this looks good.  It's a nice addition.

**Comment by roobert** [comment]
@tas50, done!

**Pull request merged**